File: q14.java
import java.util.Scanner;
public class q14{
    public static Scanner leia=new Scanner (System.in);
    public static void main (String args[]){
        int valor1,valor2,valor3,maior,maior2,resultado;

        System.out.print("Digite o primerio valor:");
        valor1=leia.nextInt();
        System.out.print("Digite o segundo valor:");
        valor2=leia.nextInt();
        System.out.print("Digite o terceiro valor:");
        valor3=leia.nextInt();
        
        if (valor1>valor2&valor1>valor3){
          maior=valor1;
          if (valor2>valor3){
            maior2=valor2;
            resultado=maior+maior2;
            System.out.println(resultado);
          }else{
            maior2=valor3;
            resultado=maior+maior2;
            System.out.println(resultado);
          }
        
        }else if (valor2>valor1&valor2>valor3){
          maior=valor2;
          if (valor1>valor3){
            maior2=valor1;
            resultado=maior+maior2;
            System.out.println(resultado);
          }else{
            maior2=valor3;
            resultado=maior+maior2;
            System.out.println(resultado);
          }
        
        }else if (valor3>valor1&valor3>valor2){
          maior=valor3;
          
          if (valor1>valor2){
            maior2=valor1;
            resultado=maior+maior2;
            System.out.println(resultado);
          }else{
            maior2=valor2;
            resultado=maior+maior2;
            System.out.println(resultado);
            }
}

        }
}

File: q14.py
valor1=int(input("Digite o primerio valor"))
valor2=int(input("Digite o segundo valor"))
valor3=int(input("Digite o terceiro valor"))

if valor1>valor2&valor1>valor3:
    maior=valor1
    if valor2>valor3:
        maior2=valor2
    else:
        maior2=valor3

elif valor2>valor1&valor2>valor3:
    maior=valor2
    if valor1>valor3:
        maior2=valor1
    else:
        maior2=valor3
elif valor3>valor2&valor3>valor1:
    maior=valor3
    if valor1>valor2:
        maior2=valor1
    else:
        maior2=valor2
else:
    ("Os valores são iguais")
print(maior)  
print(maior2)  
print(maior+maior2)    